python/hsfs/feature_view.py

```diff
@@ -101,8 +101,15 @@ def clean(feature_store_id: int, feature_view_name: str, feature_view_version: s
         )
 
     def update(self):
-        # TODO feature view: wait for RestAPI
-        return self
+        """Update the description of the feature view.
```

Comment:
Can you just add a short snippet to show that you first change the description on the object and then call the update.